README.rst

```diff
@@ -64,42 +73,43 @@ Additional optional arguments
+ password=<pwd>                   \-         \+     Specifies the password used to acquire the api-token. Use quotes if the password  contains spaces. You must specify either the password or the api_token argument. To avoid entering your password each time you run the script, use the api_token argument instead.
+ api_token=<api_token>            \-         \+     Specifies the api token for the requests; if you do not have a token, you must specify the password argument. You can retrieve your token from your profile on the **My Account** page.
+ out=<filepath>                   \+         \+     Specifies the file name, and optionally path, to which the results are written. If not specified, the default file name is ``out.csv``, written to the directory containing the script. The value of the output file must be a single .csv file that can be gzipped (extension .gz).
+ verbose                          \+         \+     Provides status updates while the script is running. It is recommended that you include this argument to track script execution progress. Silent mode (non-verbose) displays very little output.
+ keep_cols=<keep_cols>            \+         \+     Specifies the column names to append to the predictions. Enter as a comma-separated list.
+ n_samples=<n_samples>            \+         \+     DEPRECATED. Specifies the number of samples (rows) to use per batch. If not defined the "auto_sample" option will be used.
+ n_concurrent=<n_concurrent>      \+         \+     Specifies the number of concurrent requests to submit. By default, 4 concurrent requests are submitted. Set ``<n_concurrent>`` to match the number of cores in the prediction API endpoint.
+ create_api_token                 \+         \+     Requests a new API token. To use this option, you must specify the ``password`` argument for this request (not the ``api_token`` argument). Specifying this argument invalidates your existing API token and creates and stores a new token for future prediction requests.
+ n_retry=<n_retry>                \+         \+     Specifies the number of times DataRobot will retry if a request fails. A value of -1, the default, specifies an infinite number of retries.
+ pred_name=<pred_name>            \+         \+     Applies a name to the prediction column of the output file. If you do not supply the argument, the column name is blank. For binary predictions, only positive class columns are included in the output. The last class (in lexical order) is used as the name of the prediction column.
+ skip_row_id                      \+         \+     Skip the row_id column in output.
+ output_delimiter=<delimiter>     \+         \+     Specifies delimiter for output CSV. The special keyword "tab" can be used to indicate a tab delimited csv.
```

Comment:
Specifies the delimiter for the output CSV file. The special keyword "tab" can be used to indicate a tab-delimited CSV.

If you use "tab" is it still a csv (since we now accept tsv files also)? What is the default and what else can they use besides comma and tab? If they use space, for example, do they use the word or how do they enter it?